ra/ra.go

```diff
@@ -292,9 +289,9 @@ func (ra *RegistrationAuthorityImpl) NewRegistration(ctx context.Context, init c
 	// Store the authorization object, then return it
 	reg, err = ra.SA.NewRegistration(ctx, reg)
 	if err != nil {
-		// InternalServerError since the user-data was validated before being
+		// berrors.InternalServer since the user-data was validated before being
 		// passed to the SA.
-		err = core.InternalServerError(err.Error())
+		err = berrors.InternalServerError("ra.NewRegistration: %s", err)
```

Comment:
What's the purpose behind adding a method name here? Remember that the detail for InternalServerError gets passed to the user, so including method names is rather confusing.